.github/workflows/release.yml

```diff
@@ -25,13 +25,16 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-maven-master-
       - name: Setup environment
+        env:
+          DIKTAT_GPG_PUB: ${{ secrets.DIKTAT_PGP_PUB }}
+          DIKTAT_GPG_SEC: ${{ secrets.DIKTAT_PGP_SEC }}
         run: |
           echo ::set-env name=PREVIOUS_VERSION::$(printf 'VERSION=${diktat-check.version}\n0\n' | mvn help:evaluate | grep '^VERSION' | cut -d= -f2)
           echo ::set-env name=RELEASE_VERSION::${GITHUB_REF#'refs/tags/v'}
-          echo ${{ secrets.DIKTAT_GPG_PUB }} > ~/diktat.pubring.gpg
-          echo ${{ secrets.DIKTAT_GPG_SEC }} > ~/diktat.secring.gpg
+          echo "$DIKTAT_GPG_PUB" > ~/diktat.pubring.gpg
```

Comment:
Are you sure that this will not print ***

Reply:
I was afraid it would, but turns out import and signing works fine.